README.md

```diff
@@ -22,26 +22,26 @@ The AI readiness group is at the very early formation stages, as a result, the g
  - **Leverage existing frameworks** -  financial services organisations already have robust frameworks in place for dealing with technology risk. We’ll look to leverage these framework, or approaches, where appropriate.
  - **Threat modelling** - for our selected use case(s), we’ll use threat modelling techniques to map out the challenge, then identify suitable mitigation 
```

Comment:
This is pretty broad, which is fine, but might want to include that the core team is looking to further define scope.(For example, are we looking to secure the framework including data sources or just the system of record.)